CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+        - [Authors](#authors)
+    - [Peer review](#peer-review)
+    - [How to publish blog posts and deploy the blog](#how-to-publish-blog-posts-and-deploy-the-blog)
+
+<!-- /TOC -->
+
+
+## Preview the blog posts
+
+While you are developing or creating new contents for the blog, and always before publishing a PR, you should validate your changes locally, specially if you're modifying common features or you're using shortcodes.
+
+To locally serve the blog, you need to satisfy the [project requirements](README.md#requirements), and then to run:
+
+```shell
+make serve;
+```
+and then, go to [http://localhost:8484](http://localhost:8484)
+
+
+## Creating a new post
+
+Posts are stored under [`post` content directory](content/post)
+
+A blog post content stored under: `content/post/__POST_FILE_NAME__.md`
+
+will be accessible under: `//__BLOG_HOST_NAME__/post/__POST_FILE_NAME__`
+
+
+### Content schema
+
+Every blog post content file is stored under the path `content/post/__POST_FILE_NAME__.md`, and has the following schema:
+
+```
+---
+author: authorKeyName
+date: 2006-01-02
+title: "Post title"
+image: /post/__POST_FILE_NAME__/header-image.png
+description: "Short description of the post"
+categories: ["science", "technical", "culture"]
+draft: true
+---
+
+Whatever content in `markdown` format.
+
+```
+
+
+### Front matter
+
+The **front matter** is the first section inside the content file &ndash;enclosed by "`---`" chars&ndash; where it is defined the content metadata.<br />(Read more about it in the Hugo docs for the [Front Matter section](https://gohugo.io/content-management/front-matter))
+
+- `author`: Defines the authorship as explained in the [authors section](#authors)
+- `date`: Date when you plan to publish the post, `2006-01-02` format. **Do not forget to update it before publishing it.**
+- `title`: Post title, as it will appear in the very top of the content
+- `image`: Defines the content listing image as explained in the [assets section](#assets)
+- `description`: Short description of the post. Plain text only; no HTML nor markdown allowed.
+- `categories`: An array of at least one of the following: `science`, `technical`, `culture`. If it is not defined, it will not be served by the json blog api, so it will not appear in the source{d} landing.
+- `draft`: This key should not be present in public posts. Read more in the [peer review section](#peer-review)
+
+
+### Formatting the content
+
+It is not allowed to use HTML tags to format the content of the blog.
+
+If you want/need special formatting for your content, please read the [shortcodes tutorial](https://blog.sourced.tech/documentation/shortcodes)
+
+
+### Links to other source{d} blog posts
+
+Links from source{d} blog posts to other source{d} blog posts must be always relative to the blog url. That means that these urls will be like:
+```
+[link text](/post/__POST_FILE_NAME__)
+```
+
+
+### Assets
+
+Given a post content file stored under <br />
```

Comment:
are these `<br />` intentional?

Reply:
Yes, they are.
The goal is to obtain a condensed sequence instead of a expande one (that would be obtained with double line break).

![image](https://user-images.githubusercontent.com/2437584/32245462-1fda11c6-be7c-11e7-8a59-fc2e5d4a96c7.png)